### ISSUE TITLE
Create way to stop temp controller from turning off at end

### ISSUE THREAD

**Issue by harveymk** [Hendricks-Laboratory/OT2Control#21]
ETA-an hour and a half
Question to user, or implement temp_change operation that allows user to adjust temperature, and without any change the temp controller stays on?

**Comment by mphendricks** [comment]
Yeah, I think we want to implement temperature control as a line in the input sheet. We could have temp_on (plus temp_change, which would be the same as temp_on if it's already on) and temp_off as the commands, and then use a column for the input temperature. Like Max suggested, unless temp_off is called at the end, the temperature controller just stays on. 

If the temperature controller is ever turned on during the procedure, then there needs to be a check at the beginning that it's turned on and it needs to be called as a module like we currently do. If the temperature controller is not ever turned on during the procedure, then we can instead call a custom Labware object of the same shape as the temperature controller but doesn't require it to be turned on, but would allow us to use that deck position in our protocols. We have the custom json for this but it should be tested a bit more.
